Assets/Src/Scripts/Animator/HandLandmarksController.cs

```diff
@@ -87,20 +146,122 @@ public class HandLandmarksController : MonoBehaviour {
       }
     }
 
+    private void ComputeFingerRotation() {
+      _fingerTargets[0].rotation =
```

Comment:
Line 150-236:

The code block can be converted to a table-driven pattern like this:

```
var rotationTable = new (int fingerId, int landmarkId1, int landmarkId2)[] {
  (0, 2, 1),
  (1, 3, 2),
  (2, 4, 3),
  ...
};

foreach (var (fingerId, landmarkId1, landmarkId2) in rotationTable) {
  _fingerTargets[fingerId] = Quaternion.LookRotation(...
}
```